src/markspelling.py

```diff
@@ -47,17 +49,22 @@ def checkline(self, line, linenumber, filename, incodeblock=False):
         incodeblock = self.checkcodeblock(line, incodeblock)
         errorwords = list()
         if not wasincodeblock and not incodeblock:
-            self.logger.debug('Checking line "%s"', line.rstrip())
+            self.logger.debug('%03d %5s \033[1;30m%s\033[0m', linenumber, 'RAW', line.rstrip())
```

Comment:
How does this work?

Reply:
:sparkles: Magic! :sparkles:

Reply:
Should at least add a comment mentioning that they're bash escape codes.